src/kfactory/kcell.py

```diff
@@ -1464,6 +1464,7 @@ def connectivity_check(
         db: rdb.ReportDatabase | None = None,
         recursive: bool = True,
         add_cell_ports: bool = False,
+        check_layerconnectivity: bool = True,
```

Comment:
shouldn't be `check_layer_connectivity`?

Reply:
should be outdated already, but yes